wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
+
+## Recommendation 
+
+- Formalize recommendation in title 
+- Provide a brief, 1-3 sentence summary of reasoning behind this change
+- Provide alternate recommendation(s) to the one in the title if needed 
+
+## Context/Precedents 
+
+If existing, provide any research, links to PR(s) from other communities, etc. that provide precedent for this decision. 
```

Comment:
@spiffxp I looked for the framework you mentioned but couldn't find it through casual searching. I support adding it in principle!

```suggestion
Provide any research, links to PR(s) from other communities, etc. that provide precedent for this decision. 
```

Reply:
this framework https://github.com/kubernetes/community/pull/5207

which I see you've reviewed, yay